pandas/core/window.py

```diff
@@ -544,7 +544,9 @@ def _pop_args(win_type, arg_names, kwargs):
                 return all_args
 
             win_type = _validate_win_type(self.win_type, kwargs)
-            return sig.get_window(win_type, window).astype(float)
+            # GH #15662.
```

Comment:
if it were non-obvious what this parameter means / did (but passing it as ``False``). So on 2nd thought if you can provide a 1-line comment on the meaning would be good.